### ISSUE TITLE
FFMPEG failure in "Extract sequences from media files"

### ISSUE THREAD

**Issue by florianduclot** [olivierfriard/BORIS#660]
**Indicate the version of BORIS**
v8.21.7

**Describe the bug**
I see the same symptoms as described in #575.
Briefly, the process to "extract sequences from media files" appears to run without problem from the UI perspective, but "completes" very quickly and no media file is actually created in the destination folder.

**To Reproduce**
1. Click on "Observations > Extract sequences from medial files"
2. Select one observation
3. Use all defaults for other options
4. Set destination folder
5. After just a few seconds, the message "medial files created at <destination-folder>" is displayed at the bottom of BORIS' UI.

**Expected behavior**
Medial files are created in destination folder.

**Logs**
In `boris.log`, I could see that all file paths and settings were correctly set in the debug statements printing the ffmpeg command. However, there is an issue with the use of single quote (`'`) instead of double quotes (`"`).
The ffmpeg command was, for instance:
```
12:05:49,517  events_snapshots l.633 DEBUG ffmpeg command: "C:\path-to\ffmpeg.exe" -ss 1033.000 -i "path\to\sourcefile.mp4" -y -t 5.800 -an  'C:\destination-folder\filename.mp4'
```
Changing to the following works:
```
12:05:49,517  events_snapshots l.633 DEBUG ffmpeg command: "C:\path-to\ffmpeg.exe" -ss 1033.000 -i "path\to\sourcefile.mp4" -y -t 5.800 -an  "C:\destination-folder\filename.mp4"
```

**Desktop (please complete the following information):**
 - OS: windows 10
 - Type of BORIS installation: win64 installer

**Test fix**
The following line:
https://github.com/olivierfriard/BORIS/blob/28317c007a298a94f828f9c3878cd48ab2082835/boris/events_snapshots.py#L635

was changed to:
```python
                            f"{ffmpeg_command} \"{new_file_name}\"",
```

After restarting BORIS, the "Extract sequences from media files" work.

If that fix makes sense to you, let me know if you'd like me to create a PR for it.
Note that I haven't tested the "fix" in Linux.

**Comment by olivierfriard** [comment]
Hi,
thank you for your fix.
The issue (that was impacting only the Windows version) should be fixed in v. 8.21.8
Please try and let me know

**Comment by florianduclot** [comment]
Thanks a lot for the very quick fix and release, @olivierfriard.
I can confirm that this issue is now resolved in v8.21.8 so I'm closing it.
